src/pages/PoolsPage/v3/SupplyLiquidityV3/components/PresetRanges/index.tsx

```diff
@@ -0,0 +1,253 @@
+import React, { useMemo, useEffect, useState } from 'react';
+import { Layers } from 'react-feather';
+import { Currency } from '@uniswap/sdk-core';
+import { PoolStats } from '../PoolStats';
+import { IDerivedMintInfo } from 'state/mint/v3/hooks';
+import { Presets } from 'state/mint/v3/reducer';
+import { Box, Grid } from '@material-ui/core';
+import { PoolState } from 'hooks/usePools';
+import Loader from 'components/Loader';
+import { fetchPoolsAPR } from 'utils/aprApi';
+import { computePoolAddress } from 'hooks/v3/computePoolAddress';
+import { POOL_DEPLOYER_ADDRESS } from 'constants/v3/addresses';
+import './index.scss';
+
+export interface IPresetArgs {
+  type: Presets;
+  min: number;
+  max: number;
+}
+
+interface IPresetRanges {
+  mintInfo: IDerivedMintInfo;
+  baseCurrency: Currency | null | undefined;
+  quoteCurrency: Currency | null | undefined;
+  isStablecoinPair: boolean;
+  activePreset: Presets | null;
+  handlePresetRangeSelection: (preset: IPresetArgs | null) => void;
+  priceLower: string | undefined;
+  priceUpper: string | undefined;
+  price: string | undefined;
+}
+
+enum PresetProfits {
+  VERY_LOW,
+  LOW,
+  MEDIUM,
+  HIGH,
+}
+
+export function PresetRanges({
+  mintInfo,
+  baseCurrency,
+  quoteCurrency,
+  isStablecoinPair,
+  activePreset,
+  handlePresetRangeSelection,
+  priceLower,
+  price,
+  priceUpper,
+}: IPresetRanges) {
+  const [aprs, setAprs] = useState<undefined | { [key: string]: number }>();
+
+  useEffect(() => {
+    fetchPoolsAPR().then(setAprs);
+  }, []);
+
+  const ranges = useMemo(() => {
+    if (isStablecoinPair)
+      return [
+        {
+          type: Presets.STABLE,
+          title: `Stablecoins`,
+          min: 0.984,
+          max: 1.01,
+          risk: PresetProfits.VERY_LOW,
+          profit: PresetProfits.HIGH,
+        },
+      ];
+
+    return [
+      {
+        type: Presets.FULL,
+        title: `Full range`,
+        min: 0,
+        max: Infinity,
+        risk: PresetProfits.VERY_LOW,
+        profit: PresetProfits.VERY_LOW,
+      },
+      {
+        type: Presets.SAFE,
+        title: `Safe`,
+        min: 0.8,
+        max: 1.4,
+        risk: PresetProfits.LOW,
+        profit: PresetProfits.LOW,
+      },
+      {
+        type: Presets.NORMAL,
+        title: `Common`,
+        min: 0.9,
```

Comment:
we should have the ranges configurable by environment variables so we can shift the rangers easier